### PR TITLE
Prevent realtimeUnsubscribeFromVolumeIndicator error when no matching attendeeId is found

### PR DESCRIPTION
**Pull request by ltrung**
**Issue #2235 :**
`realtimeUnsubscribeFromVolumeIndicator` throw a fatal error when there are no subscriptions for the  given`attendeeId`. This results in being disconnected.

**Description of changes:**
Check that the subscription for the given `attendeeId` exists to prevent an error thrown when calling `indexOf`. There is no longer anything that would cause the try/catch to occur, so I removed that code to fix coverage.

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
No

**Checklist:**

1. Have you successfully run `npm run build:release` locally? Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/realtimecontroller/DefaultRealtimeController.test.ts [hunk 1]
```diff
@@ -548,6 +548,16 @@ describe('DefaultRealtimeController', () => {
       expect(state.volumeIndicatorCallbacks).to.equal(expectedResult);
     });
 
+    it('will not error if unsubscribed from a non-exist attendeeId', () => {
+      const callback = (
```
Comment (schontz) [inline test/realtimecontroller/DefaultRealtimeController.test.ts:552]: nit: You should be able to have a simple callback, i.e., `const callback = () => {}`.

**Review by schontz — APPROVED** [review]

File: test/realtimecontroller/DefaultRealtimeController.test.ts [hunk 2]
```diff
@@ -1195,29 +1205,6 @@ describe('DefaultRealtimeController', () => {
       expect(fatal.calledWith(matchUn)).to.be.true;
     });
 
-    it('handles broken volume callbacks', () => {
```
Comment (LeviSklaroff) [inline test/realtimecontroller/DefaultRealtimeController.test.ts:1198]: Why was this removed?

Comment (ltrung) [reply test/realtimecontroller/DefaultRealtimeController.test.ts:1198]: As commented in the PR description, there is nothing in the code that can throw error anymore so this test is no longer valid.
> There is no longer anything that would cause the try/catch to occur, so I removed that code to fix coverage.

**Review by LeviSklaroff — APPROVED** [review]
